example/stack.yaml

```diff
@@ -0,0 +1,5 @@
+flags: {}
+packages:
+- '.'
+extra-deps: []
+resolver: nightly-2015-08-21
```

Comment:
What about adding a stack version to this file? This would allow you to make sure the stack tool itself is nailed to a specific version. For repeatable builds.